charts/k8ssandra/values.yaml

```diff
@@ -169,6 +169,67 @@ stargate:
     secret: ""
     username: ""
 
+  ingress:
+    # -- Optional hostname used to match requests
```

Comment:
The `stargate.ingress.enabled` property is missing.

Reply:
>Some experience showed that layers of enabled are pretty obnoxious, and so I didn't want to keep that aspect of the structure. 

I have mixed feelings on this and it probably merits more discussion. 

>In the new setup, there is no stargate.ingress.enabled -- only stargate.ingress.[auth|rest|graphql|cassandra].enabled. They're all on by default (and, of course, they have no effect unless you've also installed an ingress controller).

As things are currently implemented though in your PR, `stargate.ingress.enabled` has to be set; otherwise, no Ingress objects will be created. You do set the property in the unit tests.

Reply:
> As things are currently implemented though in your PR, stargate.ingress.enabled has to be set; otherwise, no Ingress objects will be created. You do set the property in the unit tests.

Yep, hence draft mode. I believe it's normalized now. I'm willing to have the discussion about whether is is the right move.